### ISSUE TITLE
The method to build v.c on Linux 

### ISSUE THREAD

**Issue by shaoxie1986** [vlang/v#464]
I tested the following command:
```shell
clang -std=gnu11 -w -o vc v.c
clang -std=gnu99 -w -o vc v.c
gcc -std=gnu11 -w -o vc v.c
gcc -std=gnu99 -w -o vc v.c
```
to compile v.c, and run 
```shell
./vc -o v .
```
to compile  v, there is no crash.

os: Manjaro
gcc: 8.3.0
clang:8.0.0

maybe it can help you

**Comment by shaoxie1986** [comment]
and build with clang,when run 
```shell
./vc --help
errno=2 err='No such file or directory'
V panic: --help doesnt exist
```

with gcc 
```shell
./vc --help
[1]    3095 segmentation fault (core dumped)  ./vc --help
```

**Comment by Bowero** [comment]
This helps to compile the compiler, but it won't resolve any future bugs. You can set up `clang` as the default compiler for `cc`:

```
sudo update-alternatives --install /usr/bin/cc cc /usr/bin/clang 100
````

**Comment by whoizit** [comment]
you should launch `./v --help`, not `./vc --help`

**Comment by Xiphin** [comment]
Solved it: #507

**Comment by Oxylibrium** [comment]
Fixed with v0.1.0
